schema/collection.go

```diff
@@ -73,17 +76,18 @@ func NewDefaultCollection(name string, id uint32, schVer int, fields []*Field, i
 	// flag. Later probably we can relax it. Starting with strict validation is better than not validating extra keys.
```

Comment:
Nit: Comment could be improved. 

// Allows the caller to pass additional properties. This feature is currently disabled as it is better to be strict than to not validate and extra key. This constraint will most likely be relaxed later.